lib/cockpit-components-listing.css

```diff
@@ -1,44 +0,0 @@
-/*
- * This file is part of Cockpit.
- *
- * Copyright (C) 2016 Red Hat, Inc.
- *
- * Cockpit is free software; you can redistribute it and/or modify it
- * under the terms of the GNU Lesser General Public License as published by
- * the Free Software Foundation; either version 2.1 of the License, or
- * (at your option) any later version.
- *
- * Cockpit is distributed in the hope that it will be useful, but
- * WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public License
- * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
- */
-
-/*
- * This file contains some of the style for the listing view component in
- * cockpit-components-listing.jsx. Other rules can be found in cockpit.css in
- * the base package.
- *
- * The goal is to have all style rules specific to the listing pattern be here
- * instead of in the base package
- */
-
-div.listing-ct-head {
```

Comment:
It doesn't matter to me. Why would the CSS be react specific? Even Patternfly has made their CSS generic, and only the javascript parts have framework specific shims.